AWS/aws_redshift_get_daily_product_costs.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        # Input param validation.
</pre><p>I dont understand this runbook. Doesnt it require the customer to have redshift configured to analyze AWS costs? Is there a runbook to configure AWS to do that? I am not sure how can someone use this runbook in their environment? How should be the schema for redshift n all?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I can link to the blog posts that describe it all.

but yes - you do need a cost and usage report to be in Redshift for this to work